compatibility-verifier/sample-test-suite/pre-controller-upgrade.yaml

```diff
@@ -33,7 +39,23 @@ operations:
     tableConfigFileName: feature-test-1.json
     recordReaderConfigFileName: data/recordReaderConfig.json
     segmentName: FeatureTest1_Segment
+  - type: streamOp
+    description: create Kafka topic PinotRealtimeFeatureTest2Event
+    op: CREATE
+    streamConfigFileName: feature-test-2-realtime-stream-config.json
+  - type: streamOp
+    description: publish rows to PinotRealtimeFeatureTest2Event
+    op: PRODUCE
+    streamConfigFileName: feature-test-2-realtime-stream-config.json
+    numRows: 9
```

Comment:
Suggest publishing higher number of rows.

Reply:
the number of rows are consistent with offline data set(10 rows). Higher number of rows will need addition of more dummy data in csv and recalculate query results. What do you suggest?

Reply:
we dont need more data in csv. we loop around until we reach the number of rows, so it is ok.
yes, we do need to modify the query results.

Reply:
done!